web/src/components/forms/access-rule/components/Select.tsx

```diff
@@ -112,7 +117,11 @@ export const MultiSelect: React.FC<MultiSelectProps> = ({
             id={id}
             isDisabled={isDisabled}
             //getOptionLabel={(option) => `${option.label}  (${option.value})`}
-            options={options}
+            options={[
```

Comment:
@Eddie023 could you make this a button underneath the form field, rather than an option in the list?

Because it's an option, if the user selects it we're going to end up submitting "Select all" to our API alongside the other submitted fields.

Reply:
The `select-all` option is only shown in the frontend and won't be submitted to API.

Reply:
@chrnorm  This should work as you expect. Here is the loom video https://www.loom.com/share/172c9d95f01c4c6090fddd0f9a7ff340

Also, I personally think having a `select-all` option is a better UX 🤔

Reply:
@Eddie023 my mistake here - I read through the code without checking out your branch! Thanks for the demo video, it looks really good. Agreed on your approach here. Great work